addons/payment/models/payment_acquirer.py

```diff
@@ -98,9 +98,11 @@ def _get_providers(self, cr, uid, context=None):
     }
 
     _defaults = {
+        'provider': 'transfer',
         'company_id': lambda self, cr, uid, obj, ctx=None: self.pool['res.users'].browse(cr, uid, uid).company_id.id,
         'environment': 'test',
-        'validation': 'automatic',
+        'validation': 'manual',
+        'view_template_id': lambda self, cr, uid, obj: self.pool['ir.model.data'].get_object_reference(cr, uid, 'payment_transfer', 'transfer_acquirer_button')[1],
```

Comment:
same as above. move this to payment_transfer module and use "xmlid_to_res_id" method. get_object_reference is deprecated.